tests/serve.sh

```diff
@@ -10,3 +10,6 @@ mv src/media/js/settings_local_test.js src/media/js/settings_local.js
 make build
```

Comment:
got an itch to share this file too. also it'd be nice if we could check the webserver is up too rather than sleeping, but we don't know the TEST_URL here